docs/spec-delete.md

```diff
@@ -1,7 +1,56 @@
 ---
 id: spec-delete
-title: spec-delete
-sidebar_label: spec-delete
+title: Delete operation
 ---
 
-spec-delete TODO
+## Delete Specification
+
+Delete operation is enabled accepting single inputType as arguments
+
+For `Note` type:
+```graphql
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  comments: [Comment]!
+}
+```
+
+Following mutation will be created:
```

Comment:
Implies generation, but doesn't have to be the case if someone wants to implement this in their own API with framework.

Reply:
Amazing point